### PR TITLE
feat: new index

### PR DESCRIPTION
**Pull request by c-ehrlich**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

- new index page
- nicer styling
- invites users to check out our quick start guide and/or docs
- same tRPC and auth demos
- much shorter index.tsx files than before (42-82 lines depending on chosen packages)

The links are to the new docs, without `beta` in the URL - so don't merge until after the new docs are live!

---

## Screenshots

All packages selected
Screenshot 2022-11-08 at 21 56 46

No packages selected
Screenshot 2022-11-08 at 21 56 51

Mobile
Screenshot 2022-11-08 at 21 57 47

💯

### CODE REVIEW

**Comment by juliusmarminge** [comment]
There might be links in the readme too pointing towards the beta

File: cli/template/page-studs/index/with-auth-trpc-tw.tsx [hunk 1]
```diff
@@ -15,47 +15,40 @@ const Home: NextPage = () => {
...
+            >
+              <h3 className="text-2xl font-bold">Documentation →</h3>
+              <div className="text-lg">
+                Learn more about Create T3 App, the libraries it uses, and how
+                to deploy it.
+              </div>
+            </Link>
+          </div>
+          <div className="flex flex-col items-center gap-2">
+            <p className="text-xl text-white">
```
Comment (juliusmarminge) [inline cli/template/page-studs/index/with-auth-trpc-tw.tsx:46]: I'd up it to `text-2xl`

File: cli/template/page-studs/index/with-auth-trpc-tw.tsx [hunk 2]
```diff
@@ -72,48 +65,17 @@ const AuthShowcase: React.FC = () => {
...
-        <p className="text-2xl text-blue-500">{secretMessage}</p>
-      )}
+    <div className="flex flex-col items-center justify-center gap-4">
+      <p className="text-xl text-white">
+        {sessionData && <span>Logged in as {sessionData?.user?.name}</span>}
+        {secretMessage && <span> - {secretMessage}</span>}
+      </p>
       <button
-        className="rounded-md border border-black bg-violet-50 px-4 py-2 text-xl shadow-lg hover:bg-violet-100"
+        className="rounded-full bg-white/10 px-5 py-3 font-semibold text-white no-underline transition hover:bg-white/20"
```
Comment (juliusmarminge) [inline cli/template/page-studs/index/with-auth-trpc-tw.tsx:74]: I'd make it a bit wider, maybe `px-10`?

**Review by juliusmarminge — COMMENTED** [review]
CleanShot 2022-11-13 at 02 42 45@2x

This is `text-2xl` on the trpc message and `px-10` on the Sign In button

**Comment by c-ehrlich** [comment]
Agreed, both changes look better. I also shortened the secret message a bit

image

**Pull request merged**